engine/evalOp.cc

```diff
@@ -985,11 +996,15 @@ namespace minsky
                 {
                 case index:
                   {
-                    vector<unsigned> targetDims{unsigned(from1.rank()),unsigned(from1.numElements())};
+					vector<unsigned> targetDims;  
+                    targetDims={unsigned(from1.rank()),unsigned(from1.dataSize())};
```

Comment:
Any particular reason why you changed construction to assignment? Construction is (marginally) more efficient.

Reply:
No, I forgot to revert it to construction. This is code I had with a conditional for sparse or not. I will fix it before the next recommit.